cmd/run_konan

```diff
@@ -85,10 +85,12 @@ STUB_GENERATOR_JAR="${KONAN_HOME}/konan/lib/StubGenerator.jar"
 INTEROP_INDEXER_JAR="${KONAN_HOME}/konan/lib/Indexer.jar"
 INTEROP_JAR="${KONAN_HOME}/konan/lib/Runtime.jar"
 SHARED_JAR="${KONAN_HOME}/konan/lib/shared.jar"
+EXTRACTED_DESCRIPTORS_JAR="${KONAN_HOME}/konan/lib/konan.descriptors.jar"
```

Comment:
What does mean "extracted" here?

Reply:
Means, that it will go to Big Kotlin.

"Extracted" is a temporary module(s) and a temporary jar(s).